Create postgresql-unused-index-check.md (#80)
percona-platform/portal-doc: docs/advisors/checks/postgresql-unused-index-check.md

# Check for relations with unused indexes for PostgreSQL

## Description

> This check will scan all databases in the cluster and list relations with indexes that have not been used since the last statistics reset.

Reviewer: ```suggestion
This check scans all databases in the cluster and lists relations with indexes that have not been used since the last statistics reset.
```

## Resolution

> Connecting to a database and running the query:

Reviewer: ```suggestion
Connect to a database and run the query:
```

```
SELECT * FROM pg_stat_user_indexes 
```

> Will list the indexes for the current database. The column idx_scan will indicate howmany times the index has been used.

Reviewer: ```suggestion
The output lists the indexes for the current database. The column idx_scan indicates how many times the index has been used.
```

> If you find indexes which have not been used,  re-evaluate the need of such indexes and take appropriate actions.  Keep in mind that some indexes may be needed to address foreign key performance. For example deleting or updating a key would force postgres to validate the constraint. Without an index, this could result in sequential scans. 

Reviewer 2: ```suggestion
If you find unused indexes, check whether these indexes are needed, and take appropriate actions.  

Keep in mind that some indexes may be needed to address foreign key performance. For example deleting or updating a key would force postgres to validate the constraint. Without an index, this could result in sequential scans. 
```

> If you find indexes which have not been used, one should re-evaluate the need of the index and take appropriate action.  Keep in mind that some indexes may be needed to address foreign key performance. For example deleting or updating a key would force postgres to validate the constraint. Without an index, this could result in sequential scans. 

Reviewer: ```suggestion
If you find indexes which have not been used,  re-evaluate the need of such indexes and take appropriate actions.  Keep in mind that some indexes may be needed to address foreign key performance. For example deleting or updating a key would force postgres to validate the constraint. Without an index, this could result in sequential scans. 
```

Reviewer: Appropriate actions is too vague. I'd expect an explanation here
